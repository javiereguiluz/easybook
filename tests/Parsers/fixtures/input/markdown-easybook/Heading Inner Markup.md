SetExt Headers

Lorem `ipsum dolor` sit amet
============================

Lorem `ipsum dolor` sit amet
----------------------------

Lorem `ipsum dolor` sit **amet**
================================

Lorem `ipsum dolor` sit **amet**
--------------------------------

Lorem "ipsum dolor" sit #amet
=============================

Lorem "ipsum dolor" sit #amet
-----------------------------


Atx Headers

# Lorem `ipsum dolor` sit amet

## Lorem `ipsum dolor` sit amet

# Lorem `ipsum dolor` sit **amet**

## Lorem `ipsum dolor` sit **amet**

# Lorem "ipsum dolor" sit #amet

## Lorem "ipsum dolor" sit #amet


Atx Headers with trailing delimiters

# Lorem `ipsum dolor` sit amet #

## Lorem `ipsum dolor` sit amet #

# Lorem `ipsum dolor` sit **amet** #

## Lorem `ipsum dolor` sit **amet** #

# Lorem "ipsum dolor" sit #amet #

## Lorem "ipsum dolor" sit #amet #
