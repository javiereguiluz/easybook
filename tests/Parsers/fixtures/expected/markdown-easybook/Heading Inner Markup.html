<p>SetExt Headers</p>

<h1 id="lorem-ipsum-dolor-sit-amet">Lorem <code>ipsum dolor</code> sit amet</h1>

<h2 id="lorem-ipsum-dolor-sit-amet-2">Lorem <code>ipsum dolor</code> sit amet</h2>

<h1 id="lorem-ipsum-dolor-sit-amet-3">Lorem <code>ipsum dolor</code> sit <strong>amet</strong></h1>

<h2 id="lorem-ipsum-dolor-sit-amet-4">Lorem <code>ipsum dolor</code> sit <strong>amet</strong></h2>

<h1 id="lorem-ipsum-dolor-sit-amet-5">Lorem "ipsum dolor" sit #amet</h1>

<h2 id="lorem-ipsum-dolor-sit-amet-6">Lorem "ipsum dolor" sit #amet</h2>

<p>Atx Headers</p>

<h1 id="lorem-ipsum-dolor-sit-amet-7">Lorem <code>ipsum dolor</code> sit amet</h1>

<h2 id="lorem-ipsum-dolor-sit-amet-8">Lorem <code>ipsum dolor</code> sit amet</h2>

<h1 id="lorem-ipsum-dolor-sit-amet-9">Lorem <code>ipsum dolor</code> sit <strong>amet</strong></h1>

<h2 id="lorem-ipsum-dolor-sit-amet-10">Lorem <code>ipsum dolor</code> sit <strong>amet</strong></h2>

<h1 id="lorem-ipsum-dolor-sit-amet-11">Lorem "ipsum dolor" sit #amet</h1>

<h2 id="lorem-ipsum-dolor-sit-amet-12">Lorem "ipsum dolor" sit #amet</h2>

<p>Atx Headers with trailing delimiters</p>

<h1 id="lorem-ipsum-dolor-sit-amet-13">Lorem <code>ipsum dolor</code> sit amet</h1>

<h2 id="lorem-ipsum-dolor-sit-amet-14">Lorem <code>ipsum dolor</code> sit amet</h2>

<h1 id="lorem-ipsum-dolor-sit-amet-15">Lorem <code>ipsum dolor</code> sit <strong>amet</strong></h1>

<h2 id="lorem-ipsum-dolor-sit-amet-16">Lorem <code>ipsum dolor</code> sit <strong>amet</strong></h2>

<h1 id="lorem-ipsum-dolor-sit-amet-17">Lorem "ipsum dolor" sit #amet</h1>

<h2 id="lorem-ipsum-dolor-sit-amet-18">Lorem "ipsum dolor" sit #amet</h2>
